bundles/org.openhab.binding.heliosventilation/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="heliosventilation"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
formatting looks wrong. chan you check?

Reply:
against what should I check? I used the eclipse auto formatting Shift-Ctrl-F. Also I don't see where the format should be improved.

Reply:
If SAT is not complaining, it's ok.